File: js/spec/words_counter_spec.js
describe("Words Counter", function () {
    it('should returns 2', function () {
        expect(wordsCounter("word1 word2")).toEqual(2)
    })

    it('should returns 3', function () {
        expect(wordsCounter("word1 word2 word3")).toEqual(3)
    })

    it('should returns 2', function () {
        expect(wordsCounter(" word1 word2")).toEqual(2)
    })

    it('should returns 4',function () {
        expect(wordsCounter("word1 word2 3 word4")).toEqual(4)

    })

    it('should returns 1',function () {
        expect(wordsCounter("word1-word2")).toEqual(1)
    })

    it('should returns 2',function () {
        expect(wordsCounter("  word1  word2")).toEqual(2)
    })
});

File: spec/words_counter_spec.rb
require '../words_counter.rb'
describe WordsCounter do
  it 'should returns 2'  do
    expect(WordsCounter.count("word1 word2")).to eq 2
  end

  it 'should return 3' do
    expect(WordsCounter.count("word1 word2 word3")).to eq 3
  end

  it 'should return 2' do
    expect(WordsCounter.count(" word1 word2")).to eq 2
  end

  it 'should return 1' do
    expect(WordsCounter.count("word1-word2")).to eq 1
  end

  it 'should return4' do
    expect(WordsCounter.count("word1 word2 3 4")).to eq 4
  end

  it 'should return 2' do
    expect(WordsCounter.count("  word1  word2")).to eq 2
  end
end